javaagent/build.gradle.kts

```
@@ -271,6 +273,36 @@ with(components["java"] as AdhocComponentWithVariants) {
   }
 }
 
+spdxSbom {
+  targets {
+    // Create a target to match the published jar name.
+    // This is used for the task name (spdxSbomFor<SbomName>)
+    // and output file (<sbomName>.spdx.json).
+    create("opentelemetry-java_opentelemetry-javaagent") {
```

sorry, did I already ask about this weird naming? does it matter if we simplify it to
```suggestion
    create("opentelemetry-javaagent") {
```